docs/readme.md

```diff
@@ -0,0 +1,105 @@
+# Fixate Documentation
+
+To actually _read_ the docs, head to https://fixate.readthedocs.io. 
+
+This document is about how to add to and improve the existing docs, and the setup for building them.
+
+# Docs Structure
+
+## Theory of Documentation
+
+See [Documentation Quadrants](https://dunnhq.com/posts/2023/documentation-quadrants/). tl;dr - documentation falls into one of these categories, and it's good to both: 1. keep them separate and 2. ensure there is a good quantity/quality of each type
+
+- tutorials
+- how-to guides
+- discussions
+- reference
+
+Currently we have the following:
+
+   - quickstart (tutorial)
+   - example_walkthrough (tutorial)
+   - example projects (how-to)
+   - fixate/fixate (reference in-depth API docs, see below)
+   - release-notes (reference)
+
+# Building
+
+- Uses [Sphinx](https://www.sphinx-doc.org/)
+- `tox -e docs` will build, see `tox.ini::[testenv:docs]` section
+    - underneath it, with an appropriate venv: `fixate/docs > sphinx-build . _build`
```

Comment:
This is my last issue. It's not really clear what this line is trying to convey. Is it supposed to indicate a way to build without tox or is it supposed to be indicating what tox is doing under the hood?
If it's the former, I don't think it should be a sub-point of `tox -e docs`. If it's something else entirely, some extra clarity wouldn't go astray for the numpties like me.

Reply:
> Is it supposed to indicate a way to build without tox or is it supposed to be indicating what tox is doing under the hood?

Yeah, under the hood. Clarified now.